pkg/mod/module_test.go

```diff
@@ -54,6 +80,15 @@ func TestFindWithWrongPath(t *testing.T) {
 
 	wrongpath := "wrong_file_path/deps.sysl"
 	mod, err := Find(wrongpath, "")
-	assert.Equal(t, fmt.Sprintf("%s not found", wrongpath), err.Error())
+	assert.Error(t, err)
 	assert.Nil(t, mod)
 }
+
+func TestHasPathPrefix(t *testing.T) {
+	assert.True(t, hasPathPrefix("github.com/anz-bank/sysl", "github.com/anz-bank/sysl/deps.sysl"))
+	assert.True(t, hasPathPrefix("github.com", "github.com/anz-bank/sysl"))
+	assert.True(t, hasPathPrefix("github.com/", "github.com/anz-bank/sysl"))
+	assert.True(t, hasPathPrefix("github.com/anz-bank/sysl", "github.com/anz-bank/sysl"))
+	assert.True(t, hasPathPrefix("github.com/anz-bank/sysl", "github.com/anz-bank/sysl/"))
+	assert.False(t, hasPathPrefix("github.com/anz-bank/sysl2", "github.com/anz-bank/sysl/deps.sysl"))
+}
```

Comment:
nit: This might be improved by formatting it as a table driven test, which would allow you to give a name to each test. Totally optional though, current formatting is also okay, I had to read a few times to make sure likes 89-92 weren't duplicates